### PR TITLE
add switch libtremor

### PR DESCRIPTION
**Pull request by Cpasjuste**

### CODE REVIEW

File: switch/libtremor/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: cpasjuste <cpasjuste@gmail.com>
+
+pkgname=switch-libtremor
+pkgver=9999
```
Comment (carstene1ns) [inline switch/libtremor/PKGBUILD]: You do not really want to do this.

**Comment by WinterMute** [comment]
We have libvorbis, I don't believe we also need or want this.

The source you've used is also improperly packaged, see https://github.com/devkitPro/pacman-packages/tree/master/3ds/libvorbisidec

**Comment by Cpasjuste** [comment]
I do need it at least. I'm no going to refactor the ports using it :/

**Comment by carstene1ns** [comment]
@Cpasjuste At least tremor is nearly api compatible with ogg+vorbis(file).
Apart from build system stuff this works by just changing the headers and hardcoding a bit:
https://github.com/EasyRPG/Player/blob/master/src/decoder_oggvorbis.cpp#L119-L123

**Comment by Ghabry** [comment]
at least I observed during testing that libtremor is much faster in decoding than libvorbis.

**Comment by WinterMute** [comment]
Using correct packaging here https://github.com/devkitPro/pacman-packages/commit/ad6490250a1143d952693e22a950683593793d8d

**Pull request closed without merging**